CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # IdentityCache changelog
 
+#### 0.4.0
+
+- Return an array from fetched association to prevent chaining. Up to now, a relation was returned by default. (#287)
+
 #### 0.3.2
```

Comment:
Could we first make a 0.3.2 release?  Currently the latest on [rubygems](https://rubygems.org/gems/identity_cache) is 0.3.1